app/pages/sections/order-items/catalogue-solutions/routes.js

```diff
@@ -113,6 +113,17 @@ export const catalogueSolutionsRoutes = (authProvider, addContext, sessionManage
 
       if (apiResponse.success) {
         logger.info('redirecting catalogue solutions main page');
+        sessionManager.saveToSession({ req, key: sessionKeys.solutions, value: undefined });
+        sessionManager.saveToSession({ req, key: sessionKeys.selectedItemId, value: undefined });
+        sessionManager.saveToSession({ req, key: sessionKeys.selectedItemName, value: undefined });
+        sessionManager.saveToSession({ req, key: sessionKeys.solutionPrices, value: undefined });
+        sessionManager.saveToSession({ req, key: sessionKeys.selectedPriceId, value: undefined });
+        sessionManager.saveToSession(
+          { req, key: sessionKeys.selectedRecipients, value: undefined },
+        );
+        sessionManager.saveToSession(
+          { req, key: sessionKeys.plannedDeliveryDate, value: undefined },
+        );
```

Comment:
Would `sessionManager.clearSession` be of any use here or are there some details that need to remain in session at this point (i.e. it's too destructive)?

Reply:
I did think about that but reckon we'll need them for other bits so probs better to not get rid of everything

Reply:
Do you think it's worth extracting this to a separate helper then? Could it be useful for additional and/or associated services?

Reply:
Hm if it does get reused we should remove it but it's only used here currently
they won't have the solutions session key I don't think either